packages/millicast-sdk/src/View.js

```diff
@@ -192,6 +193,9 @@ export default class View extends BaseWebRTC {
   stop () {
     super.stop()
     this.worker?.terminate()
+    this.worker = null
+    this.codecPayloadTypeMap = {}
```

Comment:
if it's map, should use `new Map()` otherwise it's an object here

Reply:
Will change the name. It works fine as an object, and it is treated as that.